lua/bufferline/api.lua

```diff
@@ -44,6 +44,26 @@ local utils = require'bufferline.utils'
 --- @type bufferline.buffer
 local Buffer = require'bufferline.buffer'
 
+local ESC = vim.api.nvim_replace_termcodes('<Esc>', true, false, true)
+
+--- Initialize the buffer pick mode.
+--- @param fn fun()
+local function pick_buffer_wrap(fn)
```

Comment:
This is common logic among both `api.buffer_pick` _and_ `api.buffer_pick_delete`, so I extracted it into a separated function.